### DISCUSSION TITLE
Welcome to perfect-skin-website Discussions!

### DISCUSSION THREAD

**Discussion by noufbinhuwaiden** [noufbinhuwaiden/perfect-skin-website#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by noufbinhuwaiden** [comment]
hello group members what do you think about the logo 
![skin care logo](https://user-images.githubusercontent.com/116552611/199199565-6681a754-b2d6-46b3-83b5-789e48eb2ee7.PNG)
 is it suitable for our website ?

**Reply by Amal-Alwahni** [reply]
perfect !!

**Reply by EmanSalehAlyafeai** [reply]
As for me I love it !!! 
If anyone has a better suggestion share it with us, otherwise if you all agree on this one then it's decided

**Reply by RenadAlMokbily** [reply]
yes great!!

**Reply by Maryam3333** [reply]
> hello group members what do you think about the logo
> ![skin care logo](https://user-images.githubusercontent.com/116552611/199199565-6681a754-b2d6-46b3-83b5-789e48eb2ee7.PNG)
> is it suitable for our website ?

Great

**Comment by noufbinhuwaiden** [comment]
![Product2](https://user-images.githubusercontent.com/116552611/199199835-70fd4a52-a608-4cc1-87bc-3c84bd81feca.png)
![Product3](https://user-images.githubusercontent.com/116552611/199199847-ad44fb25-c418-4329-b2d5-7cecea223f1e.png)
![Product4](https://user-images.githubusercontent.com/116552611/199199852-83b804ea-17fa-4815-8364-43d86e2daeff.png)
 what do you think about these products as interface ??

**Reply by EmanSalehAlyafeai** [reply]
Great! For the products in the products page, I added one more product which is the 'la roche posay effaclar duo acne spot treatment' and wrote about it

**Reply by Amal-Alwahni** [reply]
& I chose a product from "Glow Recipe"!!

**Comment by noufbinhuwaiden** [comment]
![main-image3](https://user-images.githubusercontent.com/116552611/199199991-40bd0994-f5a9-4327-8542-16e7d422db99.png)

is the model suitable for our interface?

**Reply by Amal-Alwahni** [reply]
Beautiful !!

**Comment by Amal-Alwahni** [comment]
![GlowReci](https://user-images.githubusercontent.com/113019419/199201377-3362d833-0ada-4502-a7da-c1c49de786af.jpeg)

So I chose this product to discuss about for my product page.

**Comment by Amal-Alwahni** [comment]
should we choose one of your products as our sub product or we can choose anything on our own?

**Reply by EmanSalehAlyafeai** [reply]
I'm not sure , how many sub-products there will be? 5? 
I think each one of us can choose her own, as for me I chose another product as a sub one, which I mentioned about below Nouf's comment ^

**Reply by Amal-Alwahni** [reply]
alright !

**Comment by Amal-Alwahni** [comment]
girls we will submit individually isnt it?

**Reply by RenadAlMokbily** [reply]
yes submit the two pages you created

**Reply by Maryam3333** [reply]
Yes

**Comment by Amal-Alwahni** [comment]
Dis everyone submit their work?

**Reply by Amal-Alwahni** [reply]
did*

**Comment by Amal-Alwahni** [comment]
i linked each of our pages with my pages so when you click any button on nav it'll take you to another page

**Comment by Amal-Alwahni** [comment]
[Website.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/9916957/Website.zip)

**Comment by Amal-Alwahni** [comment]
everyone send your zip final file here!

**Reply by RenadAlMokbily** [reply]
[Website.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/9917470/Website.zip)

**Reply by EmanSalehAlyafeai** [reply]
[Website.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/9946438/Website.zip)

**Comment by Maryam3333** [comment]
I will send it now

**Comment by Maryam3333** [comment]
[Website.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/9917350/Website.zip)

**Comment by Amal-Alwahni** [comment]
girls i used this background for my product page
![watercolor2](https://user-images.githubusercontent.com/113019419/201098448-db877f2c-5c19-4c8d-8bcc-445530a05332.jpg)

**Comment by Amal-Alwahni** [comment]
once everyone finishes plz send me the links of your page so i can link them to mine

**Comment by Maryam3333** [comment]
Sure, Amal.

**Comment by Amal-Alwahni** [comment]
hey girls
i have finished my javascript part and ill send it so let me know how do you think of it.
does it need any changes or its good?

**Comment by Amal-Alwahni** [comment]
[Uploading term2022_project_AlwahniAmal_U19105519.zip…]()

**Comment by RenadAlMokbily** [comment]
this is phase 3 
[renad phase 3.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/10022050/renad.phase.3.zip)

**Comment by EmanSalehAlyafeai** [comment]
[term2022_project_stage3_Alyafeai_U20103890.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/10024928/term2022_project_stage3_Alyafeai_U20103890.zip)

**Comment by Amal-Alwahni** [comment]
so girls for phase 4 JavaScript i chose for one page to be clock and dark mode and for the other page just clock and alert box for discount code is that ok?

**Reply by RenadAlMokbily** [reply]
yes its great

**Comment by Amal-Alwahni** [comment]
![1](https://user-images.githubusercontent.com/113019419/202464275-f76e2b2c-55a5-4378-bb07-34d912c171e8.jpg)
![2](https://user-images.githubusercontent.com/113019419/202464287-91aa4980-7
![3](https://user-images.githubusercontent.com/113019419/202464301-055f6b66-0070-4cc3-89c2-73c7e112b1b5.jpg)
b6b-447c-bed2-88e06c930aa4.jpg)

**Comment by Amal-Alwahni** [comment]
![2](https://user-images.githubusercontent.com/113019419/202464445-63321b45-c9c4-4a66-bd83-864a0d00389a.jpg)
![4](https://user-images.githubusercontent.com/113019419/202464480-109339f0-f373-4b77-9977-1e6a86d61dae.jpg)

**Comment by Amal-Alwahni** [comment]
good morning girls, what did you choose for the JavaScript ?

**Reply by EmanSalehAlyafeai** [reply]
Hii Amal, I chose to create a countdown timer using javascript that counts how many days are left to some event we'll host, like starting free online skincare courses or something similar! I didn't finish it yet, it'll be ready by today's evening inshallah!

**Reply by RenadAlMokbily** [reply]
hi, 
in my part, the program asks the user to input his/her name. then, at the center top of the page a welcome text pops with the name of the user.

**Comment by Maryam3333** [comment]
what I did for the JavaScript part in my pages is, change the background color using JavaScript, and a WELCOME alert using JavaScript.

**Comment by Amal-Alwahni** [comment]
That's great! so, once we submit our individual parts, we will then discuss on at least 2 JavaScript to attach it to GitHub.

**Comment by Maryam3333** [comment]
Hello,
We have to choose two JavaScripts to attach it to GitHub, so what do you prefer to choose?

**Comment by Amal-Alwahni** [comment]
So, we'll be choosing the (Time) JavaScript and the (Discount) JavaScript

**Reply by Maryam3333** [reply]
ok

**Comment by Amal-Alwahni** [comment]
<script>
setInterval(myTimer, 1000);

function myTimer() {
  const d = new Date();
  document.getElementById("demo").innerHTML = d.toLocaleTimeString();
}
</script>

</body>

**Comment by EmanSalehAlyafeai** [comment]
[term2022_project_stage4_Alyafeai_U20103890.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/10097869/term2022_project_stage4_Alyafeai_U20103890.zip)

**Comment by RenadAlMokbily** [comment]
[Project_stage4_almokbily_u20102793.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/10097893/Project_stage4_almokbily_u20102793.zip)

**Comment by Maryam3333** [comment]
[Website.zip](https://github.com/noufbinhuwaiden/perfect-skin-website/files/10098610/Website.zip)
This is my part for phase 4, please let me know if it needs any modification.

**Reply by EmanSalehAlyafeai** [reply]
Nice! but wait I think it doesn't show that you replaced the extra products with what we chose
